File: Modul-1/C/PRAK105-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
    int a, b, x, y;

    printf("Vaiabel a bernilai: ");
    scanf("%d", &a);
    printf("Vaiabel b bernilai: ");
    scanf("%d", &b);
    printf("Vaiabel x bernilai: ");
    scanf("%d", &x);
    printf("Vaiabel y bernilai: ");
    scanf("%d", &y);

    int sisaBagiA = a % b;
    int sisaBagiX = x % y;
    int totalSisaBagi = sisaBagiA + sisaBagiX;

    printf("Total sisa bagi dari a dibagi b dan x dibagi y adalah %d\n", totalSisaBagi);

    return 0;
}

File: Modul-1/python/PRAK105-2310817110006-Adrian_Bintang_Saputera.py
a = int(input("Variabel a bernilai: "))
b = int(input("Variabel b bernilai: "))
x = int(input("Variabel x bernilai: "))
y = int(input("Variabel y bernilai: "))

sisa_bagi_a = a % b
sisa_bagi_x = x % y
total_sisa_bagi = sisa_bagi_a + sisa_bagi_x

print(f"Total sisa bagi dari a dibagi b dan x dibagi y adalah {total_sisa_bagi}")
